static/js/components/InstrumentSummary.jsx

```diff
@@ -96,8 +65,19 @@ const InstrumentSummary = ({ route }) => {
   // Load the instrument if needed
   useEffect(() => {
     dispatch(Action.fetchInstrument(route.id));
-    setStartDate(defaultStartDate);
-    setEndDate(defaultEndDate);
+
+    setLoading(true);
+    dispatch(
+      Action.fetchInstrumentLogs(route.id, {
+        startDate: defaultStartDate,
+        endDate: defaultEndDate,
+      }),
+    ).then((response) => {
+      if (response.status !== "success") {
+        dispatch(showNotification("Error fetching instrument logs", "error"));
+      }
+      setLoading(false);
```

Comment:
No it doesn't, that's correct. However I would prefer having a dedicated PR for that. I refactored/wrote some of that code to avoid existing issues but any other improvement would deserve a PR of their own.

Reply:
That works, please file an issue.